### ISSUE TITLE
Remnants left over after pip uninstall

### ISSUE THREAD

**Issue by AustEcon** [electrumsv/electrumsv-sdk#7]
After running the `electrumsv-sdk start` command it clones the repositories to site-packages

But `pip uninstall` does not remove this so I am left with:

![image](https://user-images.githubusercontent.com/8123976/88055171-a28d2e00-cbb2-11ea-9252-ef2f84cb536a.png)

Not sure right at the moment how to deal with this other than perhaps making the *first ever* run of electrumsv-sdk purge all of this leftover cruft from prior installations... not ideal though because if I move those directories around in future it could get awkward.

I use the `Manifest.in` to include arbitrary things in site-packages...and seems to be cleaned up alright on pip uninstall...
But can't find any docs about removing things that are not present at the time of creating the build and dist for pypi...
in other words all of the extra stuff that gets generated at run-time of the SDK is not removed with a pip uninstall.

Maybe there is a way but haven't found any help for it yet.

**Comment by rt121212121** [comment]
Shouldn't you be putting user data in AppData like the ESV data? 

```
from os.path import expanduser
home = expanduser("~")
```

**Comment by AustEcon** [comment]
I guess that's one way... There'd be no way (that I know of) to make pip uninstall find it and remove it so I'd go with the same concept - i.e. do a search for pre-existing remains of prior RegTest SDK installs and nuke it for the very first run.
- It's maybe a bit inelegant but it should get the job done

**Comment by rt121212121** [comment]
I think you should be keeping the runs as sessions and the user should have the option to potentially copy a session and continue from where it left off. Or put it aside and come back to it while starting afresh for something else.

**Comment by AustEcon** [comment]
I think this is sorted now to my satisfaction with version 0.0.13 - LMK if you run into any issues with testing it.
